themes/default/content/registry/packages/exoscale/installation-configuration.md

```diff
@@ -21,8 +21,8 @@ To provision resources with the Pulumi Exoscale provider, you need to provide th
 
 Use `pulumi config set exoscale:<option>`.
 
-| Option | Required? | Description |
-| - | - | - |
-| `key` | Required | Exoscale account API key (environment: `EXOSCALE_API_KEY`). |
-| `secret` | Required | Exoscale account API secret (environment: `EXOSCALE_API_SECRET`). |
-| `timeout` | Optional | Global async operations waiting time in seconds (default: 300). |
+| Option    | Required/Optional | Description                                                       |
+|-----------|-------------------|-------------------------------------------------------------------|
+| `key`     | Required          | Exoscale account API key (environment: `EXOSCALE_API_KEY`).       |
+| `secret`  | Required          | Exoscale account API secret (environment: `EXOSCALE_API_SECRET`). |
+| `timeout` | Optional          | Global async operations waiting time in seconds (default: 300).   |
```

Comment:
| Option | Required? | Description |
| - | - | - |
| `key` | Required | Exoscale account API key (environment: `EXOSCALE_API_KEY`). |
| `secret` | Required | Exoscale account API secret (environment: `EXOSCALE_API_SECRET`). |
| `timeout` | Optional | Global async operations waiting time in seconds (default: 300). |